web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala

```diff
@@ -96,9 +96,23 @@ import scala.util.{Failure, Success}
     )
 
     val error = formData.firstValidationError.map { text =>
+      val emailNotVerified = "The email is not verified, check your spam folder if you don't see the email."
```

Comment:
This will work but it's not ideal. Be sure to add a TODO so we can fix  it later